### PR TITLE
Use alternate name for DATABASE_PORT var

### PR DESCRIPTION
**Pull request by danieltharp**

### CODE REVIEW

**Comment by danieltharp** [comment]
This also renames the github actions file to include `php` as there will be another build one for postgres dev. This PR should be ready to squash and merge.

File: .github/workflows/docker-build-push-php.dev.yml [hunk 1]
```diff
@@ -31,7 +32,7 @@ jobs:
             IMAGE_TAG: ${{ github.sha }}
           run: |
             set -ex
-            docker build --build-arg MAIN_DOMAIN=wikijump.dev --build-arg FILES_DOMAIN=wjfiles.dev -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+            docker build --build-arg WIKIJUMP_REPO_BRANCH=${GITHUB_REF##*/} --build-arg MAIN_DOMAIN=wikijump.dev --build-arg FILES_DOMAIN=wjfiles.dev -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
```
Comment (rossjrw) [inline .github/workflows/docker-build-push-php.dev.yml:35]: Is `${GITHUB_REF##*/}` a shell variable? If so:

1. what does `##*/` mean?
1. I don't see where `GITHUB_REF` is being set - should this be an actions expression, e.g. [`${{ github.ref }}`](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context)?

Comment (danieltharp) [reply .github/workflows/docker-build-push-php.dev.yml:35]: `GITHUB_REF` is an environment variable set by the GitHub Actions runner that contains the branch along with some other stuff, the `##*/` is a substring search in bash: https://tldp.org/LDP/abs/html/parameter-substitution.html

Comment (rossjrw) [reply .github/workflows/docker-build-push-php.dev.yml:35]: > ${var##Pattern} Remove from $var the longest part of $Pattern that matches the front end of $var. 

I assume it takes `refs/heads/whatever`, globs `*/` against `refs/heads/` and trims it? If so, works for me

**Review by rossjrw — APPROVED** [review]

**Pull request merged**